### ISSUE TITLE
QA for 0.4.2

### ISSUE THREAD

**Issue by apyrgio** [freedomofpress/dangerzone#458]
To ensure that new releases do not introduce regressions, and support existing and newer platforms, we have to run some QA tests. Copied from [RELEASE.md](https://github.com/freedomofpress/dangerzone/blob/main/RELEASE.md):

- [ ] In `.circleci/config.yml`, add new platforms and remove obsolete platforms
- [ ] Make sure that the tip of the `main` branch passes the CI tests.
- [ ] Create a test build in Windows and make sure it works:
  - [ ] Create a new development environment with Poetry.
  - [ ] Build the container image and ensure the development environment uses
    the new image.
  - [ ] Run the Dangerzone tests.
  - [ ] Build and run the Dangerzone .exe
  - [ ] Test some QA scenarios (see [Scenarios](#Scenarios) below).
- [ ] Create a test build in macOS (Intel CPU) and make sure it works:
  - [ ] Create a new development environment with Poetry.
  - [ ] Build the container image and ensure the development environment uses
    the new image.
  - [ ] Run the Dangerzone tests.
  - [ ] Create and run an app bundle.
  - [ ] Test some QA scenarios (see [Scenarios](#Scenarios) below).
- [ ] Create a test build in macOS (M1/2 CPU) and make sure it works:
  - [ ] Create a new development environment with Poetry.
  - [ ] Build the container image and ensure the development environment uses
    the new image.
  - [ ] Run the Dangerzone tests.
  - [ ] Create and run an app bundle.
  - [ ] Test some QA scenarios (see [Scenarios](#Scenarios) below).
- [ ] Create a test build in the most recent Ubuntu LTS platform (Ubuntu 22.10
  as of writing this) and make sure it works:
  - [ ] Create a new development environment with Poetry.
  - [ ] Build the container image and ensure the development environment uses
    the new image.
  - [ ] Run the Dangerzone tests.
  - [ ] Create a .deb package and install it system-wide.
  - [ ] Test some QA scenarios (see [Scenarios](#Scenarios) below).
- [ ] Create a test build in the most recent Fedora platform (Fedora 37 as of
  writing this) and make sure it works:
  - [ ] Create a new development environment with Poetry.
  - [ ] Build the container image and ensure the development environment uses
    the new image.
  - [ ] Run the Dangerzone tests.
  - [ ] Create an .rpm package and install it system-wide.
  - [ ] Test some QA scenarios (see [Scenarios](#Scenarios) below).

We will track our progress on this issue by checking items in the above list.

_(see also the previous [QA for 0.4.1](https://github.com/freedomofpress/dangerzone/issues/350))_

**Comment by deeplow** [comment]
When a document fails it's saying the same as in the terminal (`UNTRUSTED>`). Maybe we can make this more appealing to users visually?

![badmood](https://github.com/freedomofpress/dangerzone/assets/47065258/cb73792a-4cf5-4879-8daf-817bfd7108e8)

Maybe `[UNTRUSTED]`. But still it feels odd to see it in the UI like this. Maybe in the future we should add some styling. And in any case this will probably go away since we're moving to client-side error messages for the most part.

**Comment by apyrgio** [comment]
Yeah, I didn't pour much thought into this, since we want to avoid showing possibly-attacker controlled messages to the user in the first place. In any case, `[UNTRUSTED]` vs `UNTRUSTED>` makes little difference to me to warrant a change now. If think it's important though, feel free to send a PR and I'll approve it.

**Comment by apyrgio** [comment]
I haven't managed to test the following:

> Create an .rpm with ./install/linux/build-rpm.py package and install it in a fedora qube and make sure conversions run with containers

due to lack of space and time. But that's something that we have already tested through various way in other steps, so I'm pretty confident that it will work.

**Comment by deeplow** [comment]
Closing this one since it's now been released.
